docs/core/rid-catalog.md

```diff
@@ -80,11 +80,13 @@ For the latest and complete version, see the [runtime.json](https://github.com/d
 
 .NET Core 2.0 SDK introduces the concept of portable RIDs. They are new values added to the RID graph that aren't tied to a specific version or OS distribution. They're particularly useful when dealing with multiple Linux distros.
 
-The following list shows the most common RIDs used for each OS. It doesn't cover `arm` or `corert` values.
+The following list shows a small subset of the most common RIDs used for each OS.
 
 ## Windows RIDs
 
-- Portable
+Only common values are listed. For the latest and complete version, see the [runtime.json](https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.NETCore.Platforms/runtime.json) file on CoreFX repo.
+
+- Portable (.NET Core 2.0 or later versions)
   - `win-x86`
   - `win-x64`
```

Comment:
Add `win-arm` and `win-arm64`

Reply:
Added and swapped win-x86 to be below win-x64 to follow other sections which prioritized x64 on top.